globus_cli/config_command/edit.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import print_function
+import click
+import os
+import subprocess
+
+from globus_cli.helpers import common_options
+
+
+@click.command('edit', help='Edit your Globus Config file')
+@common_options(no_format_option=True)
+def edit_command():
+    """
+    Executor for `globus config edit`
+    """
+    editor = os.environ.get('EDITOR', 'nano')
+    subprocess.call([editor, os.path.expanduser('~/.globus.cfg')])
```

Comment:
I don't know if the SDK will even work on Windows yet.
I'm not sure that it's a target platform for the CLI?

Until we have a more clearly established stance on Windows (and I have a testing setup for it), I'm not sure I want to add windows-specific features.